### DISCUSSION TITLE
How to use APM in packages without prefix: air. ?

### DISCUSSION THREAD

**Discussion by megajogos** [airsdk/apm#116] [Q&A]
We have apps without the prefix: air. but the apm always put the prefix air. 

How to avoid this prefix?

**Comment by marchbold** [comment]
`apm` will use the normal methods to determine if the air prefix should be used ie:

- Checks for the `AIR_NOANDROIDFLAIR` env variable
- Checks if the current AIR SDK `adt.cfg` has the `AddAirToAppID` property set

Are you using the same environment and air sdk to build your app as you are for apm?
